### DISCUSSION TITLE
How do I forward the current panel?

### DISCUSSION THREAD

**Discussion by air71818** [Stellarium/stellarium#3679] [Q&A] [answered]
<img width="803" alt="截圖 2024-03-21 下午9 40 36" src="https://github.com/Stellarium/stellarium/assets/30951473/8d4fc0cc-96e7-446d-96f5-47228ef6a4f5">

When I using api mode, how do I get this panel?

**Comment by gzotti** [comment]
There is a main program window, and an optional web browser GUI/HTTP API interface. There is no additional graphics display via web browser.

**Reply by air71818** [reply]
Hi @gzotti, thanks for reply, so could I see the above graph via HTTP API?

**Reply by gzotti** [reply]
No. There is no additional graphics display via web browser and no image transport via HTTP API because there is a main program window where the graphics is shown. There is also Spout output if so configured. There are ways to push Spout through TCP/IP connections like Spout2NDI, but this is beyond scope.

**Reply by air71818** [reply]
So if I would like to create the little web service to have the below functional like
`react UI <------> fastapi service <------> local stellarium`
which way you recommend to fetch the above graph to UI 🤔

**Reply by gzotti** [reply, marked as answer]
Use a different program like Stellarium Web? Or sometimes you must do your own research.
